src/NzbDrone.Core/ImportLists/LazyLibrarian/LazyLibrarianImport.cs

```diff
@@ -10,7 +11,7 @@ public class LazyLibrarianImport : HttpImportListBase<LazyLibrarianImportSetting
         public override string Name => "LazyLibrarian";
 
         public override ImportListType ListType => ImportListType.Other;
-
+        public override TimeSpan MinRefreshInterval => TimeSpan.FromHours(12);
```

Comment:
Can bring this down to align with `Readarr` Implementation given it's user hosted and not an external site.